Won't compile if @types/youtube is not included in package.json

Using `"angular-youtube-player": "^1.0.2"`
Specifically, the following error occurs when building the application:
```
ERROR in node_modules/angular-youtube-player/lib/ytplayer.component.d.ts(8,26): error TS2503: Cannot find namespace 'YT'.
node_modules/angular-youtube-player/lib/ytplayer.component.d.ts(1,23): error TS2688: Cannot find type definition file for 'youtube'.
```
I managed to resolve this by adding `"@types/youtube": "^0.0.35"` to my `package.json` under `devDependencies`, but since this wasn't documented anywhere I figured I should report this bug.

--- a/projects/slot-machine-button/package.json
+++ b/projects/slot-machine-button/package.json
@@ -1,9 +1,9 @@
 {
   "name": "ng-slot-machine-button",
-  "version": "0.0.1",
+  "version": "1.0.0-alpha.1",
   "peerDependencies": {
-    "@angular/common": "^6.0.0",
-    "@angular/core": "^6.0.0",
+    "@angular/common": ">=6",
+    "@angular/core": ">=6",
     "gsap": "^2.0.0"
   },
   "description": "An Angular component module with slot machine button effects",

--- a/projects/ytplayer/package.json
+++ b/projects/ytplayer/package.json
@@ -1,9 +1,10 @@
 {
   "name": "angular-youtube-player",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "peerDependencies": {
-    "@angular/common": "^6.0.0",
-    "@angular/core": "^6.0.0"
+    "@angular/common": ">=6",
+    "@angular/core": ">=6",
+    "@types/youtube": "0.0.38"
   },
   "description": "An Angular component module for using the embedded YouTube iframe player.",
   "repository": {

